diffeqzoo/ivps.py

```diff
@@ -874,3 +874,61 @@ def oregonator(
         initial_values=initial_values,
         time_span=time_span,
     )
+
+
+def goodwin(
+    *,
+    initial_values=(
+        0.0,
+        0.0,
+    ),
+    time_span=(0.0, 25.0),
+    parameters=(
+        10,
+        1.0,
+        3.0,
+        0.5,
+        1.0,
+    ),
```

Comment:
Maybe the indentation/linebreaks could be made a little more space-efficient:

```suggestion
    *,
    initial_values=(0.0, 0.0),
    time_span=(0.0, 25.0),
    parameters=(10, 1.0, 3.0, 0.5, 1.0),
    ),
```